modules/stanford_events_importer/tests/src/Unit/Plugin/Field/FieldWidget/StanfordEventsImporterAPIURLFieldWidgetTest.php

```diff
@@ -156,6 +166,8 @@ public function setup() {
 
     // Make it.
     $this->plugin = new StanfordEventsImporterAPIURLFieldWidget($plugin_id, $plugin_definition, $field_definition, $settings, $third_party_settings, $cache->reveal());
+    $this->client = new Client();
```

Comment:
all guzzle clients should be mock objects in tests.

Reply:
http://docs.guzzlephp.org/en/stable/testing.html

☝️  That was so much help.

Reply:
interesting. I didn't have to do all that for the cap tests. :shrug:

Reply:
https://github.com/SU-SWS/stanford_person/blob/8.x-1.x/modules/stanford_person_importer/tests/src/Unit/CapTest.php#L61